Files to be translated to Hindi (Group 4)

SaranjeetKaur opened the pull request.

SaranjeetKaur committed:
add error_log

SaranjeetKaur committed:
add files to be translated to Hindi

SaranjeetKaur committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
translate 1 message

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
translate R-AllS4-hi_IN.po

jyoti-bhogal committed:
translate R-shift-hi_IN.po

jyoti-bhogal committed:
translate R-timetaken-hi_IN.po

jyoti-bhogal committed:
translate R-cedta-hi_HI.po

jyoti-bhogal committed:
translate R-fdroplevels-hi_IN.po

jyoti-bhogal committed:
:books: translate R-duplicated-hi_IN.po

jyoti-bhogal committed:
:books: translate R-tables-hi_IN.po

jyoti-bhogal committed:
:books: translate R-as.data.table-hi_IN.po

jyoti-bhogal committed:
books: translate R-merge-hi_IN.po

amansingh1011 commented on hindi_files/R-AllS4-hi_IN.po:
```diff
@@ -0,0 +1,15 @@
+
+#: AllS4.R:2
+msgid "data.table package loaded. When developing don't load package"
+msgstr "data.table पैकेज लोड हो गया है। विकास करते समय पैकेज लोड न करें।"
```
```suggestion
msgstr "data.table पैकेज (package) लोड हो गया है। विकास करते समय पैकेज (package) लोड न करें।"
```

amansingh1011 commented on hindi_files/R-as.data.table-hi_IN.po:
```diff
@@ -0,0 +1,62 @@
+
+#: as.data.table.R:90
+msgid "Argument 'sorted' must be scalar logical and non-NA"
+msgstr "तर्क 'sorted' स्केलर (scalar) लॉजिकल (logical) और गैर-NA होना चाहिए"
```
```suggestion
msgstr "तर्क 'sorted' स्केलर (scalar) लॉजिकल (logical) और non-NA होना चाहिए"
```

amansingh1011 commented on hindi_files/R-as.data.table-hi_IN.po:
```diff
@@ -0,0 +1,62 @@
+"Argument 'value.name' must be scalar character, non-NA and at least one "
+"character"
+msgstr ""
+"तर्क 'value.name' स्केलर (scalar) कैरेक्टर (character) होना चाहिए, नॉन-NA और कम से कम एक कैरेक्टर (character) होना चाहिए"
```
```suggestion
"तर्क 'value.name' स्केलर (scalar) कैरेक्टर (character) होना चाहिए, non-NA और कम से कम एक कैरेक्टर (character) होना चाहिए"
```

jyoti-bhogal committed:
Apply suggestions from Aman's code review

Co-authored-by: Aman Singh <78440410+amansingh1011@users.noreply.github.com>

jyoti-bhogal committed:
update translation of 'length'

jyoti-bhogal committed:
correct translation for message no. 32

jyoti-bhogal committed:
remove extra fullstop mark

SaranjeetKaur committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
translate R-as.data.table-hi_IN.po

jyoti-bhogal committed:
translate R-test.data.table-hi_IN.po

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
translate R-test.data.table-hi_IN.po

jyoti-bhogal committed:
translate R-test.data.table-hi_IN.po

jyoti-bhogal committed:
translate R-test.data.table-hi_IN.po

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
translate R-setkey-hi_HI.po

jyoti-bhogal committed:
translate R-setkey-hi_HI.po

jyoti-bhogal committed:
translate R-setkey-hi_HI.po

jyoti-bhogal committed:
translate R-setkey-hi_HI.po

jyoti-bhogal committed:
translate R-setkey-hi_HI.po

jyoti-bhogal committed:
translate R-setkey-hi_HI.po

jyoti-bhogal committed:
translate R-setkey-hi_HI.po

jyoti-bhogal committed:
translate cj-hi_IN.po

jyoti-bhogal committed:
translate types-hi_IN.po

jyoti-bhogal committed:
translate fcast-hi_IN.po

jyoti-bhogal committed:
translate fastmean-hi_IN.po

jyoti-bhogal committed:
translate openmp-utils-hi_IN.po

jyoti-bhogal committed:
translate fifelse-hi_IN.po

jyoti-bhogal committed:
translate fifelse-hi_IN.po

jyoti-bhogal committed:
translate fifelse-hi_IN.po

jyoti-bhogal commented:
@SaranjeetKaur I am requesting your review of the commits I made to this PR (#6571 ).

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal commented on hindi_files/R-AllS4-hi_IN.po:
```diff
@@ -0,0 +1,15 @@
+
+#: AllS4.R:2
+msgid "data.table package loaded. When developing don't load package"
+msgstr "data.table पैकेज (package) लोड हो गया है। विकास करते समय पैकेज (package) लोड न करें।"
```
```suggestion
msgstr "data.table पैकेज (package) लोड (load) हो गया है। विकास करते समय पैकेज (package) लोड (load) न करें।"
```

jyoti-bhogal commented on hindi_files/R-as.data.table-hi_IN.po:
```diff
@@ -0,0 +1,62 @@
+"as.data.table.array method should only be called for arrays with 3+ "
+"dimensions; use the matrix method for 2-dimensional arrays"
+msgstr ""
+"as.data.table.array विधि केवल 3+ आयामों वाले एरे (array) के लिए ही कॉल की जानी चाहिए; "
```
```suggestion
"as.data.table.array विधि केवल 3+ आयामों वाले एरे (array) के लिए ही कॉल (call) की जानी चाहिए; "
```

jyoti-bhogal commented on hindi_files/R-as.data.table-hi_IN.po:
```diff
@@ -0,0 +1,62 @@
+"Argument 'value.name' must be scalar character, non-NA and at least one "
+"character"
+msgstr ""
+"तर्क 'value.name' स्केलर (scalar) कैरेक्टर (character) होना चाहिए, non-NA और कम से कम एक कैरेक्टर (character) होना चाहिए"
```
```suggestion
"तर्क 'value.name' स्केलर (scalar) कैरेक्टर (character), non-NA और कम से कम एक कैरेक्टर (character) का होना चाहिए"
```

jyoti-bhogal commented on hindi_files/R-as.data.table-hi_IN.po:
```diff
@@ -0,0 +1,62 @@
+
+#: as.data.table.R:108
+msgid "Argument 'value.name' should not overlap with column names in result: %s"
+msgstr "तर्क 'value.name' परिणाम में कॉलम (column) नामों के साथ परस्पर नहीं होना चाहिए: %s"
```
```suggestion
msgstr "तर्क 'value.name' परिणाम में कॉलम (column) नाम के साथ परस्पर नहीं होना चाहिए: %s"
```

jyoti-bhogal commented on hindi_files/R-as.data.table-hi_IN.po:
```diff
@@ -0,0 +1,62 @@
+
+#: as.data.table.R:181
+msgid "Item %d has %d rows but longest item has %d; recycled with remainder."
+msgstr "आइटम (item) %d में %d पंक्तियाँ हैं लेकिन सबसे लंबे आइटम (item) में %d है; शेष के साथ पुनः चक्रित किया गया।"
```
```suggestion
msgstr "आइटम (item) %d में %d रोज़ (rows) हैं लेकिन सबसे लंबे आइटम (item) में %d है; शेष के साथ पुनः चक्रित किया गया।"
```

jyoti-bhogal commented on hindi_files/R-cedta-hi_IN.po:
```diff
@@ -0,0 +1,19 @@
+"cedta decided '%s' wasn't data.table aware. Here is call stack with [[1L]] "
+"applied:"
+msgstr ""
+"cedta ने निर्णय लिया कि '%s' data.table के प्रति जागरूक नहीं था। यहां कॉल (call) स्टैक (stack) है [[1L]] के साथ"
```
```suggestion
"cedta ने निर्णय लिया कि '%s' data.table के प्रति जागरूक नहीं था। यह कॉल (call) स्टैक (stack) [[1L]] के साथ"
```

jyoti-bhogal commented on hindi_files/R-fdroplevels-hi_IN.po:
```diff
@@ -0,0 +1,17 @@
+msgid ""
+"droplevels() with in.place=TRUE is deprecated. Use setdroplevels() instead."
+msgstr ""
+"droplevels() के साथ in.place=TRUE को अब उपयोग नहीं किया जाएगा। इसके स्थान पर setdroplevels() का उपयोग करें।"
```
```suggestion
"droplevels() के साथ in.place=TRUE का अब उपयोग नहीं किया जाएगा। इसके स्थान पर setdroplevels() का उपयोग करें।"
```

jyoti-bhogal commented on hindi_files/R-merge-hi_IN.po:
```diff
@@ -0,0 +1,67 @@
+
+#: merge.R:37
+msgid "A non-empty vector of column names is required for `by.x` and `by.y`."
+msgstr "`by.x` और `by.y` के लिए एक गैर-खाली कॉलम (column) नामों का वेक्टर (vector) आवश्यक है।"
```
```suggestion
msgstr "`by.x` और `by.y` के लिए एक गैर-खाली कॉलम (column) नाम का वेक्टर (vector) आवश्यक है।"
```

jyoti-bhogal commented on hindi_files/R-merge-hi_IN.po:
```diff
@@ -0,0 +1,67 @@
+
+#: merge.R:52
+msgid "A non-empty vector of column names for `by` is required."
+msgstr "`by` के लिए एक गैर-खाली कॉलम (column) नामों का वेक्टर (vector) आवश्यक है।"
```
```suggestion
msgstr "`by` के लिए एक गैर-खाली कॉलम (column) नाम का वेक्टर (vector) आवश्यक है।"
```

jyoti-bhogal commented on hindi_files/R-setkey-hi_IN.po:
```diff
@@ -0,0 +1,182 @@
+"the original data's order by group. Try setindex() instead. Or, set*(copy(."
+"SD)) as a (slow) last resort."
+msgstr ""
+".SD पर एक भौतिक कुंजी सेट (set) करना भविष्य के संभावित उपयोग के लिए आरक्षित है; समूह के अनुसार मूल डेटा (data) के आदेश को संशोधित करने के लिए।"
```
```suggestion
".SD पर एक भौतिक कुंजी सेट (set) करना भविष्य के संभावित उपयोग के लिए आरक्षित है; समूह के अनुसार मूल डेटा (data) के क्रम को परिवर्तित करने के लिए।"
```

jyoti-bhogal commented on hindi_files/R-setkey-hi_IN.po:
```diff
@@ -0,0 +1,182 @@
+
+#: setkey.R:47
+msgid "cols is the empty string. Use NULL to remove the key."
+msgstr "cols खाली स्ट्रिंग है। कुंजी हटाने के लिए NULL का उपयोग करें।"
```
```suggestion
msgstr "cols खाली स्ट्रिंग (string) है। कुंजी हटाने के लिए NULL का उपयोग करें।"
```

jyoti-bhogal commented on hindi_files/R-setkey-hi_IN.po:
```diff
@@ -0,0 +1,182 @@
+"Column '%s' is type '%s' which is not supported as a key column type, "
+"currently."
+msgstr ""
+"कॉलम (column) '%s' का प्रकार '%s' है जो वर्तमान में कुंजी कॉलम (column) प्रकार के रूप में समर्थित नहीं है।"
```
```suggestion
"कॉलम (column) '%s' का प्रकार '%s' है जो वर्तमान में मुख्य कॉलम (column) प्रकार के रूप में समर्थित नहीं है।"
```

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

SaranjeetKaur committed:
Merge branch 'master' into hindi_j

SaranjeetKaur commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+"Argument #%d is of type %s, however argument #2 is of type %s. Please make "
+"sure all output values have the same type."
+msgstr ""
+"तर्क #%d प्रकार %s का है, हालाँकि तर्क #2 प्रकार %s का है।"
```
```suggestion
"आर्गुमेंट (argument) #%d प्रकार %s का है, हालाँकि आर्गुमेंट (argument) #2 प्रकार %s का है।"
```

SaranjeetKaur commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+"Argument #%d has different class than argument #2, Please make sure all "
+"output values have the same class."
+msgstr ""
+"तर्क #%d की तर्क #2 से अलग क्लास (class) है,"
```
```suggestion
"आर्गुमेंट (argument) #%d की आर्गुमेंट (argument) #2 से अलग क्लास (class) है,"
```

SaranjeetKaur commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+msgid ""
+"Argument #2 and argument #%d are both factor but their levels are different."
+msgstr ""
+"तर्क #2 और तर्क #%d दोनों फ़ैक्टर (factor) हैं लेकिन उनके स्तर (levels) भिन्न हैं।"
```
```suggestion
"आर्गुमेंट (argument) #2 और आर्गुमेंट (argument) #%d दोनों फ़ैक्टर (factor) हैं लेकिन उनके स्तर (levels) भिन्न हैं।"
```

SaranjeetKaur commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+"sure all output values have the same type."
+msgstr ""
+"तर्क #%d प्रकार %s का है, हालाँकि तर्क #2 प्रकार %s का है।"
+"कृपया सुनिश्चित करें कि दोनों तर्कों का प्रकार समान हो।"
```
```suggestion
"कृपया सुनिश्चित करें कि सभी आउटपुट (output) मान का प्रकार समान हो।"
```

SaranjeetKaur commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+"output values have the same class."
+msgstr ""
+"तर्क #%d की तर्क #2 से अलग क्लास (class) है,"
+"कृपया सुनिश्चित करें कि दोनों तर्कों का प्रकार समान हो।"
```
```suggestion
"कृपया सुनिश्चित करें कि सभी आउटपुट (output) मान की क्लास (class) समान हो।"
```

SaranjeetKaur commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+"both arguments have the same class."
+msgstr ""
+"परिणामी मान की 'डिफ़ॉल्ट' (default) से अलग क्लास (class) है।"
+"कृपया सुनिश्चित करें कि दोनों तर्कों का प्रकार समान हो।"
```
```suggestion
"कृपया सुनिश्चित करें कि दोनों आर्गुमेंट्स (arguments) की क्लास (class) समान हो।"
```

SaranjeetKaur commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+#: fifelse.c:300
+#, c-format
+msgid "Length of 'default' must be 1 or %lld."
+msgstr "'डिफ़ॉल्ट' (default) की लेंथ (length) 1 या %lld होनी चाहिए।"
```
```suggestion
msgstr "'डिफ़ॉल्ट' ('default') की लेंथ (length) 1 या %lld होनी चाहिए।"
```

SaranjeetKaur commented on hindi_files/openmp-utils-hi_IN.po:
```diff
@@ -0,0 +1,66 @@
+#: openmp-utils.c:84
+#, c-format
+msgid "  OpenMP version (_OPENMP)       %d\n"
+msgstr "  OpenMP संस्करण (_OPENMP)     %d\n"
```
```suggestion
msgstr "  OpenMP वर्जन (version) (_OPENMP)     %d\n"
```

SaranjeetKaur commented on hindi_files/openmp-utils-hi_IN.po:
```diff
@@ -0,0 +1,66 @@
+#: openmp-utils.c:100
+#, c-format
+msgid "  data.table is using %d threads with throttle==%d. See ?setDTthreads.\n"
+msgstr "  data.table %d थ्रेड्स का उपयोग कर रहा है जिसमें throttle==%d है। अधिक जानकारी के लिए ?setDTthreads देखें।\n"
```
```suggestion
msgstr "  data.table %d थ्रेड्स (threads) का उपयोग कर रहा है जिसमें throttle==%d है। अधिक जानकारी के लिए ?setDTthreads देखें।\n"
```

SaranjeetKaur commented on hindi_files/openmp-utils-hi_IN.po:
```diff
@@ -0,0 +1,66 @@
+
+#: openmp-utils.c:114
+msgid "'throttle' must be a single number, non-NA, and >=1"
+msgstr "'throttle' एक एकल संख्या, non-NA, और >=1 होना चाहिए।"
```
```suggestion
msgstr "'throttle' एक single संख्या, non-NA, और >=1 होना चाहिए।"
```

SaranjeetKaur commented on hindi_files/openmp-utils-hi_IN.po:
```diff
@@ -0,0 +1,66 @@
+
+#: openmp-utils.c:128
+msgid "threads= must be either NULL or a single number >= 0. See ?setDTthreads."
+msgstr "threads= या तो NULL होना चाहिए या एक एकल संख्या >= 0। अधिक जानकारी के लिए ?setDTthreads देखें।"
```
```suggestion
msgstr "threads= या तो NULL होना चाहिए या एक single संख्या >= 0। अधिक जानकारी के लिए ?setDTthreads देखें।"
```

SaranjeetKaur reviewed (commented):
Thanks @jyoti-bhogal for your contribution! Excellent work!

I've added some minor suggestions.

jyoti-bhogal committed:
Apply suggestions from code review

Applying code review suggestions.

Co-authored-by: Saranjeet Kaur <kaur.saranjeet3@gmail.com>

jyoti-bhogal committed:
Apply suggestions from code review

Applying suggestions from code review. Also edited a few of the suggestions. For example, change `single` to `सिंगल (single)` and a few more.

Co-authored-by: Saranjeet Kaur <kaur.saranjeet3@gmail.com>

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

amansingh1011 committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal committed:
Merge branch 'master' into hindi_j

jyoti-bhogal commented on hindi_files/fifelse-hi_IN.po:
```diff
@@ -0,0 +1,200 @@
+#: fifelse.c:199
+#, c-format
+msgid "Type '%s' is not supported"
+msgstr "प्रकार '%s' का समर्थन नहीं किया जाता है।"
```
```suggestion
msgstr "प्रकार '%s' का समर्थन नहीं किया जाता है"
```

jyoti-bhogal commented on hindi_files/R-AllS4-hi_IN.po:
```diff
@@ -0,0 +1,15 @@
+
+#: AllS4.R:2
+msgid "data.table package loaded. When developing don't load package"
+msgstr "data.table पैकेज (package) लोड (load) हो गया है। विकास करते समय पैकेज (package) लोड (load) न करें।"
```
```suggestion
msgstr "data.table पैकेज (package) लोड (load) हो गया है। विकास करते समय पैकेज (package) लोड (load) न करें"
```

jyoti-bhogal commented on hindi_files/R-setkey-hi_IN.po:
```diff
@@ -0,0 +1,182 @@
+
+#: setkey.R:206
+msgid "Internal code should not be being called on type double"
+msgstr "आंतरिक कोड (code) को प्रकार डबल (double) पर नहीं चलाना चाहिए।"
```
```suggestion
msgstr "आंतरिक कोड (code) को प्रकार डबल (double) पर नहीं चलाना चाहिए"
```

jyoti-bhogal commented on hindi_files/R-test.data.table-hi_IN.po:
```diff
@@ -0,0 +1,180 @@
+"**** इस R सेशन (session) की भाषा अंग्रेज़ी नहीं है। प्रत्येक परीक्षण अभी भी यह जाँच करेगा"
+"कि त्रुटियों और/या\n"
+"**** चेतावनियों की सही संख्या उत्पन्न होती है। हालाँकि, प्रत्येक त्रुटि/चेतावनी के पाठ का परीक्षण करने के लिए"
+"कृपया LANGUAGE=en के साथ R को पुनः प्रारंभ करें।"
```
```suggestion
"कृपया LANGUAGE=en के साथ R को पुनः प्रारंभ करें"
```

jyoti-bhogal committed:
Apply suggestions from code review

Co-authored-by: Jyoti Bhogal <bhogaljyoti1@gmail.com>
